pkg/scheduler/testing/wrappers.go

```diff
@@ -477,6 +478,17 @@ func (n *NodeWrapper) Label(k, v string) *NodeWrapper {
 	return n
 }
 
+// Taint add a taint to node with the current time.
+func (n *NodeWrapper) Taint(k, v, e string) *NodeWrapper {
```

Comment:
Prefer to pass-in a `TaintEffect` type as the 3rd parameter.

Reply:
yep, bad code.